Makefile

```diff
@@ -129,6 +141,18 @@ kind-install:
 	ENABLE_SSL=true dist/images/install.sh
 	kubectl describe no
 
+.PHONY: kind-install-cluster
+kind-install-cluster:
+	kind load docker-image --name kube-ovn $(REGISTRY)/kube-ovn:$(RELEASE_TAG)
+	kind load docker-image --name kube-ovn1 $(REGISTRY)/kube-ovn:$(RELEASE_TAG)
+	kubectl config use-context kind-kube-ovn
+	ENABLE_SSL=true dist/images/install.sh
+	kubectl describe no
+	kubectl config use-context kind-kube-ovn1
+	sed -e 's/10.16.0/10.18.0/g' -e 's/10.96.0/10.98.0/g' -e 's/100.64.0/100.68.0/g'  dist/images/install.sh > /install-multi.sh
+	ENABLE_SSL=true bash /install-multi.sh
```

Comment:
1. Place `install-multi.sh` in current directory to avoid permission issues.
2. Add `install-multi.sh` to `.gitignore`.

Reply:
added.